samples/testapp/src/commonMain/kotlin/org/multipaz/testapp/App.kt

```diff
@@ -336,6 +341,12 @@ class App private constructor (val promptModel: PromptModel) {
         }
     }
 
+    @OptIn(ExperimentalResourceApi::class)
+    private suspend fun faceMatchLiteRtModelInit() {
+        val modelData = ByteString(*Res.readBytes("files/facenet_512.tflite"))
+        faceMatchLiteRtModel = FaceMatchLiteRtModel(modelData, 160, 512)
```

Comment:
Please use named parameters so it's clear what 160 and 512 means (yes, Android Studio might have a hint but people could be reading the source from GitHub / code-review / etc).